PHPAdvanced/Task_1/Api.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+require 'requests.php';
+
+/**
+ * This is API class, we make object for each section and store it's corresponding value and implement that.
+ */
+class API{
+  private string $domain_name = 'https://www.innoraft.com';
+  private string $main_url = 'https://www.innoraft.com/jsonapi/node/services';
+  private $data = array();
+  public $field_secondary_title;
+  public $field_image;
+  public $field_service;
+  public $alias;
+  public $icon_array = [];
+  public $child_count;
+
+  /**
+   * This is a constructor, set all the at the time of creation of an object.
+   *
+   * @param  int $i
+   *  Section number.
+   *
+   * @return void
+   */
+  function __construct ($i) {
+    $response = request($this->main_url);
```

Comment:
Can't we make the request once, store the data in the object and re-use it for different values of $i?

Reply:
Now, I am using static variable for storing data and share among all objects.

Reply:
Now, I used static variable to store once and share with everyone.